<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientSearchDialog</class>
 <widget class="QDialog" name="ClientSearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поиск пациента</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="lblAddInfo">
         <property name="text">
          <string>Дополнительная информация:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="txtAddInfo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CTableView" name="tblClients">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>2</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblClientsCount">
         <property name="text">
          <string>Список пуст</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Фильтр</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <widget class="QCheckBox" name="chkFilterLastName">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>14</height>
           </size>
          </property>
          <property name="text">
           <string>Фамилия</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLineEdit" name="edtFilterLastName">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QCheckBox" name="chkFilterFirstName">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>14</height>
           </size>
          </property>
          <property name="text">
           <string>Имя</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QCheckBox" name="chkFilterPatrName">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>14</height>
           </size>
          </property>
          <property name="text">
           <string>Отчество</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QLineEdit" name="edtFilterPatrName">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="0" colspan="2">
         <widget class="QDialogButtonBox" name="filterButtonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLineEdit" name="edtFilterFirstName">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QCheckBox" name="chkFilterBirthDate">
          <property name="text">
           <string>Дата рождения</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QDateEdit" name="edtFilterBirthDate">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QCheckBox" name="chkFilterBirthDateEnd">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>по</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QDateEdit" name="edtFilterBirthDateEnd">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="accelerated">
           <bool>false</bool>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QCheckBox" name="chkFilterSex">
          <property name="text">
           <string>Пол</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QComboBox" name="cmbFilterSex">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>М</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ж</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
